src/state_machine.zig

```diff
@@ -136,6 +136,13 @@ pub fn StateMachineType(
             Storage,
             AccountImmutable,
             .{
+                .ids = .{
+                    .timestamp = 1,
```

Comment:
Suggestion:

Since it's a stable id for the lsm schema, it can be useful to reference those constants from somewhere that isn't a specialized type.

Just an example, it may not be the best place:

```zig
pub const AccountImmutable = extern struct {
    pub const schema_ids = .{
        .timestamp = 1,
        .id = 2,
        .user_data = 3,
        // ...
    };
}
```

The downside is losing the nice sight of all increasing ids declared together, but anyway, it can become unordered if we evolve our schema in the future.

Reply:
Great idea!